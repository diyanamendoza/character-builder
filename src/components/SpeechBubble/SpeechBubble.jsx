import React from 'react'

export default function SpeechBubble({ mottos }) {
  return (
    <div className="speechbubble">
      {mottos.map((motto) => (
        <h3
          key={motto + Math.random() * 10}
          style={{
            backgroundImage: 'url(./speechbubble.png)',
            backgroundRepeat: 'no-repeat',
            width: 200,
            height: 125,
            color: 'black',
            display: 'flex',
            flexDirection: 'column',
            justifyContent: 'center',
            alignItems: 'flex-start',
            overflowWrap: 'anywhere',
            paddingLeft: 80,
            paddingRight: 70,
          }}
        >
          {motto}
        </h3>
      ))}
    </div>
  )
}
