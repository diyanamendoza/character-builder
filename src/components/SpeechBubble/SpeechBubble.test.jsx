import { render } from '@testing-library/react';
import SpeechBubble from './SpeechBubble.jsx';

it('should render speech bubble component', () => {
  const { container } = render(
    <SpeechBubble mottos={['hi', 'get outta here']} />
  );
  expect(container).toMatchSnapshot();
});
