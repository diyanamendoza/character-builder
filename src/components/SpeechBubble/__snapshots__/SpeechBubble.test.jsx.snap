// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render speech bubble component 1`] = `
<div>
  <div
    class="speechbubble"
  >
    <h3
      style="background-image: url(./speechbubble.png); background-repeat: no-repeat; width: 200px; height: 125px; color: black; display: flex; flex-direction: column; justify-content: center; align-items: flex-start; overflow-wrap: anywhere; padding-left: 80px; padding-right: 70px;"
    >
      hi
    </h3>
    <h3
      style="background-image: url(./speechbubble.png); background-repeat: no-repeat; width: 200px; height: 125px; color: black; display: flex; flex-direction: column; justify-content: center; align-items: flex-start; overflow-wrap: anywhere; padding-left: 80px; padding-right: 70px;"
    >
      get outta here
    </h3>
  </div>
</div>
`;
